pb/sensorsinfo_proto/sensorsinfo.proto

```diff
@@ -81,8 +81,7 @@ message Sensor{
 message App{
     int32 UID = 1;
     string PackageName = 2;
-    repeated SubscriptionInfo Subscriptions = 3;
-    repeated ActiveConn ActiveConns = 4;
+    repeated string SensorActivities = 3;
```

Comment:
What is the motivation behind this change?

Reply:
The motivation is to avoid storing unnecessary information.
To implement the filtering-by-app feature, we just need to know what sensors have related activities to this app. Such information can be simply captured by recording the sensor name, rather than the actual events or active connections corresponding to the app. 
In this case, the filtering-by-app feature is fairly simple: we can do a first filtering to only keep the sensors whose name has shown in the list. Then only show events whos UID matches the app we are looking at.